### ISSUE TITLE
Editor on dblclick in multiselect mode

### ISSUE THREAD

**Issue by zinvob** [telerik/kendo-ui-core#5663]
### Bug report

Your sample kendo-ui-core/docs/knowledge-base/grid-click-selection-batch-edit.md
does not work properly with focus on cell.

Cell focus does not work.

### Reproduction of the problem
Start 
https://docs.telerik.com/kendo-ui/knowledge-base/grid-click-selection-batch-edit
in dojo.
Notice that there is only focus on the sell while holding down the mouse key, after release it's gone. This gives problems when working on a grid with columns outside view. I need to click inside the grid and be able to use arrow keys to get to the cell I want, and the doubleclick. Or even better, press enter to get into edit mode.

### Current behavior
Single click select row, but have no focus inside grid.
### Expected/desired behavior
Single click select row, and have focus on the cell you clicked.
### Environment

Latest version of Kendo UI Grid JQuery, all browsers etc.

**Comment by dmanova** [comment]
You are right we missed focusing the cell. To do that after `grid.closeCell(); `the following statement should be added:
`grid.table.focus();`
